doc/how-to-write-conformance-tests.md

```diff
@@ -60,28 +62,68 @@ pub fn test_my_feature<T: MyTrait>() {
 ### Step 3: Instantiate Tests for SUT Types
 
 Use the fully qualified path to the parent test module and
-`instantiate_conformance_tests!` to run the tests for each SUT type.
+`instantiate_conformance_tests!` or `instantiate_conformance_tests_with_module!`
+to run the tests for each SUT type.
 
 Assuming the tests are part of a `my_test_crate` crate, which contains
-the modules `my_module1` and `my_module2`, the tests would be instantiated as
-follows:
+the modules `my_module`, the tests would be instantiated as follows:
 
 ```rs
 use iceoryx2_bb_testing::instantiate_conformance_tests;
+use my_impl::{SUT1, SUT2, SUT3};
 
 mod sut1_impl {
-    super::instantiate_conformance_tests!(my_test_crate::my_module1, my_impl::SUT1);
-    super::instantiate_conformance_tests!(my_test_crate::my_module2, my_impl::SUT1);
+    use super::*;
+    instantiate_conformance_tests!(my_test_crate::my_module, super::SUT1);
 }
 
 mod sut2_impl {
-    super::instantiate_conformance_tests!(my_test_crate::my_module1, my_impl::SUT2);
-    super::instantiate_conformance_tests!(my_test_crate::my_module2, my_impl::SUT2);
+    use super::*;
+    instantiate_conformance_tests!(my_test_crate::my_module, super::SUT2);
 }
 
 mod sut3_impl {
-    super::instantiate_conformance_tests!(my_test_crate::my_module1, my_impl::SUT3);
-    super::instantiate_conformance_tests!(my_test_crate::my_module2, my_impl::SUT3);
+    use super::*;
+    instantiate_conformance_tests!(my_test_crate::my_module, super::SUT3);
+}
+```
+
+For such simple cases, the `instantiate_conformance_tests_with_module!` macro
```

Comment:
What is a "simple" case? Maybe elaborate on the scenario where the following fits best.

Reply:
I tried something. Not sure if it got better